ksp_plugin/interface.cpp

```diff
@@ -89,6 +93,33 @@ namespace {
 int const chunk_size = 64 << 10;
 int const number_of_chunks = 8;
 
+FixedStepSizeIntegrator<Ephemeris<Barycentric>::NewtonianMotionEquation> const&
+ParseFixedStepSizeIntegrator(std::string const& integrator_kind) {
```

Comment:
This is called "make" for the existing methods, so `MakeFixedStepSizeIntegrator`, etc.

Reply:
No, since this just parses a string field. Could have `Make` functions taking the `ConfigurationParameters`.